### PR TITLE
RequestLog - added source for logging 

### PR DESCRIPTION
**Pull request by nireeshT**
**Affects:** \<hygieia-core-version-number>.

---

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/util/CommonConstants.java [hunk 1]
```diff
@@ -4,4 +4,5 @@
 
     public static final String HEADER_CLIENT_CORRELATION_ID = "Client-Correlation-ID";
     public static final String HEADER_API_USER = "apiUser";
+    public static final String HEADER_SOURCE = "source-application";
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/util/CommonConstants.java:7]: This is not required

**Review by rvema — CHANGES_REQUESTED** [review]
Please address review comments

File: src/main/java/com/capitalone/dashboard/util/CommonConstants.java (hunk 1 above)
Comment (nireeshT) [reply src/main/java/com/capitalone/dashboard/util/CommonConstants.java]: done

**Review by rvema — APPROVED** [review]
lgtm

File: src/main/java/com/capitalone/dashboard/model/RequestLog.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
     private int responseCode;
     private long timestamp;
     private long responseTime;
+    private String source;
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/model/RequestLog.java]: is it a good idea to rename the field as `application` ?

**Review by rvema — APPROVED** [review]
lgtm

**Pull request merged**